bin/yaml/cpp.yaml

```diff
@@ -438,6 +438,8 @@ compilers:
             - 4.10.0
             - name: 4.11.1
               url: https://github.com/kalray/build-scripts/releases/download/v{name}/gcc-kalray-kvx-ubuntu-20.04-v{name}.tar.gz
+            - name: 4.12.0
+              url: https://github.com/kalray/build-scripts/releases/download/v{name}/gcc-kalray-kvx-ubuntu-20.04-v{name}.tar.gz
```

Comment:
I wonder if we could not create a new group instead of c/p the url field here... But that's ok for now, thanks!

Reply:
Yes, perhaps we can do it the day we decide to drop some obsolete releases. Is it common that old compilers get dropped on other arch ?